app/code/Magento/Customer/Test/Mftf/ActionGroup/StoreFrontClickSignInButtonActionGroup.xml

```diff
@@ -1,15 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<!--
- /**
-  * Copyright © Magento, Inc. All rights reserved.
-  * See COPYING.txt for license details.
-  */
--->
-
-<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-        xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
-    <actionGroup name="StorefrontClickSignInButtonActionGroup">
```

Comment:
Please, take a look at `app/code/Magento/Customer/Test/Mftf/Test/PasswordAutocompleteOffTest.xml:50`. This action group is used within the scope of the mentioned test. Why it's considered as redundant?

Reply:
The same situaiton. I'm removing `StoreFront`, you are mentioning use of `Storefront` :man_facepalming:

Reply:
Hi @lbajsarowicz 
@rogyar actually have a point here. If you have a reference in test on some action group, it will not rely on File name it will rely on action group `name` attribute. 
You can have `BlaBlaBlaActionGroup.xml` with content:
```
    <actionGroup name="StorefrontClickSignInButtonActionGroup">
        .......
    </actionGroup>
```

And in test you will reference this action group like `<actionGroup ref="StorefrontClickSignInButtonActionGroup" stepKey="storeFrontClickSignInButton"/>` and this will work.

To resolve the issue we should just rename this action group file to be the same as action group `name` attr.
I believe in this example someone commit `StoreFront` then fixed it to `Storefront` in file name and attr, but because someones filesystem is case insensitive this change applied only to attribute.